.github/workflows/triage.yml

```diff
@@ -24,11 +24,14 @@ jobs:
           ref: develop # DO NOT CHANGE
 
       - uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065
+      - uses: ./.github/actions/checkout-spack
 
       - name: Install Python dependencies
         run: pip install -r .github/workflows/requirements/triage/requirements.txt
 
-      - run: python3 .github/workflows/bin/spack-labeler.py
```

Comment:
Maybe split these into two steps with different names. Easier to search the logs when each function is  its own section.